code/modules/shuttle/emergency.dm

```diff
@@ -180,7 +180,16 @@
 					continue
 				if(isanimal(player)) //Poly does not own the shuttle
 					continue
-
+				if(ishuman(player)) //hostages allowed on the shuttle, check for restraints
```

Comment:
I think you can do the following here and remove both the `issilicon` and `isanimal` checks, as neither of these will be humans
```
if(!ishuman(player))
    continue
```

Reply:
this would change how hijack behaves as currently xenos will prevent a hijack unless I'm mistaken. this is consistent with hijack's objective description that pets and borgs are allowed, benos are neither of those